akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -654,15 +654,7 @@ private[http] object HttpServerBluePrint {
           f()
       }
 
-      /*
-       * WebSocket support
-       */
-      def switchToWebSocket(handlerFlow: Either[Graph[FlowShape[FrameEvent, FrameEvent], Any], Graph[FlowShape[Message, Message], Any]]): Unit = {
```

Comment:
Good cleanup here. It was a bit weird anyway that part of the WebSocket flow creation was done inside here.